### ISSUE TITLE
[pulseaudio] Cannot use sink-input thing because of configuration misbehavior

### ISSUE THREAD

**Issue by dalgwen** [openhab/openhab-addons#11272]
## Expected Behavior
The sink-input thing in the pulseaudio binding should be working, online, etc, when properly configured.
The documentation should explain how to activate it.
Also, the binding configuration should be available in the UI

## Current Behavior
The sink-input thing can be created but is always shown in error state, no matter the configuration.
The binding requires a parameter to activate the parsing of sink-input entries on the pulseaudio server. This is unfortunately undocumented but could be read in the code. BUT, even with the parameter activated in a service cfg file like this : 

> binding.pulseaudio:sinkInput=true

The thing doesn't work because of a configuration issue : the binding get the parameter as string and the cast to boolean throw an exception.

> java.lang.ClassCastException: class java.lang.String cannot be cast to class java.lang.Boolean (java.lang.String and java.lang.Boolean are in module java.base of loader 'bootstrap')

~~Also, the gear icon in the "add thing / choose binding" page doesn't show up. So far, to configure the binding, the only solution is to use a text file.~~ PR 1134 close this

## Possible Solution
I will provide a PR, which will do the following thing : 
- ~~Activate the GUI for the binding configuration by moving an xml description file inside the binding description file~~ Already done in PR 1134
- Patch the configuration code of the binding to properly load value
- Add a section in the documentation to warn users they should activate a parameter to use certain type of thing 

## Steps to Reproduce (for Bugs)
Create a sink-input thing --> always on error state
Eventually try to configure the binding to activate the parsing of sink-input on the pulseaudio server with a configuration file ---> class cast exception

**Comment by lolodomo** [comment]
You can configure the binding with the UI. This is not necessarily obvious to find but click on the binding in MainUI and you will find a way to its configuration.

**Comment by lolodomo** [comment]
Read the discussion in #11034 to understand how to find the config in UI

**Comment by dalgwen** [comment]
OK, I found your commit, thanks.
It seems we found the same issue and the same correction (moving the xml configuration description inside the binding.xml file).
I should have checked with the dev branch instead of sticking with 3.1, it could have saved me some time...

Anyway, if I'm not mistaken, the configuration by file is still broken ? (because of the class cast exception) Or maybe I configure it wrong, as I'm not familiar with text based configuration. Is there a way to force a boolean type for the parameter valuer ?
Others bindings, like the network binding, have not this issue. My test shows that it's because they used a different way of getting the parameters value.

I will provide a PR (it's nearly ready, with other things), at least for this cast exception when using text configuration, and also for the documentation.
